jetstream/sponsored-suggestion-placement-phase-2.toml

```diff
@@ -0,0 +1,694 @@
+[experiment]
+
+enrollment_period = 14
+end_date = '2024-04-22'
+segments = ["cohort_1", "cohort_2"] # Two cohorts
+
+## Segments
+
+[segments]
+[segments.cohort_1]
+select_expression = "enrollment_date BETWEEN '2024-01-17' AND '2024-01-30'"
+data_source = "segment_enrollments"
+
+[segments.cohort_2]
+select_expression = "enrollment_date BETWEEN '2024-01-17' AND '2024-02-06'"
```

Comment:
Was this one intended to be overlapping with cohort 1? or disjoint? Right now it overlaps

Reply:
Oh yes. I'll disconnect them.

One more thing, if I want to get a read on the entire population enrolled in the experiment, will that come automatically, or do I need to specify and additional cohort? If I do this, will `enrollment_period` be the maximum of enrollment periods across all cohorts?

Reply:
That'll be automatic, it'll be the `all` segment. And yes, set `enrollment_period` to cover all the cohorts. So if you have 4 weekly cohorts, set `enrollment_period` to 28 and you'll get 5 segments (`all`, `cohort_1`, ...)